torch/_subclasses/fake_tensor.py

```diff
@@ -329,7 +344,7 @@ def to_copy(fake_mode, func, *args, **kwargs):
 
     input_device = new_kwargs.pop("device", None)
     out_device = input_device if input_device else new_kwargs["input"].device
-    with no_dispatch(), in_kernel_invocation_manager(fake_mode):
+    with in_kernel_invocation_manager(fake_mode):
```

Comment:
What's going on here?

Reply:
This is because I moved no_dispatch within `in_kernel_invocation_manager`, as you suggested